ilastik/applets/pixelClassification/pixelClassificationDataExportGui.py

```diff
@@ -48,8 +41,15 @@ def __init__(self, *args, **kwargs):
         super(PixelClassificationResultsViewer, self).__init__(*args, **kwargs)
         self.topLevelOperatorView.PmapColors.notifyDirty(bind(self.updateAllLayers))
         self.topLevelOperatorView.LabelNames.notifyDirty(bind(self.updateAllLayers))
+        self._childOps = []
 
     def setupLayers(self):
+        for op in self._childOps:
+            # self._initPredictionLayers may have created an op for our Gui purposes.
+            # Clean up before potentially creating another one.
+            op.Input.disconnect()
+            op.cleanUp()
```

Comment:
is there a reason not to remove the reference once cleaned up so that it can be gc'ed?